packages/protractor/src/protractor.conf.js

```diff
@@ -43,6 +44,49 @@ function setConf(conf, name, value, msg) {
   conf[name] = value;
 }
 
+/**
+ * Helper function to find a particular namedFile
+ * within the webTestMetadata webTestFiles
+ */
+function findNamedFile(webTestMetadata, key) {
```

Comment:
is this protractor change related to cross-platform build?

Reply:
Initially yes as I thought the tests needed to be set to `extract = "run"` and protractor didn't handle the extract browser archive case; but it turned out I could get it working with `extract = "build"` in the angular repo so I can factor the protractor change out into a separate PR if you prefer. It already is a separate commit.

Reply:
yeah I'd rather have it in a separate PR

Reply:
👍